airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -27,6 +27,7 @@
 from airflow.models import BaseOperator
 from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
 from airflow.providers.amazon.aws.hooks.logs import AwsLogsHook
+from airflow.providers.amazon.aws.models.exceptions import ECSOperatorError
```

Comment:
```suggestion
from airflow.providers.amazon.aws.exceptions import ECSOperatorError
```

I think would be better name.

Reply:
Agreed.  Changed.